src/patternfly/components/Form/styles.scss

```diff
@@ -199,178 +149,48 @@ $pf-c-form__select--m-error-svg--Viewbox: "0 0 26 15.8" !default;
-      background-color: var(--pf-c-form__input--disabled--BackgroundColor);
-      border-color: var(--pf-c-form__input--disabled--BorderColor);
-      @include pf-bg-svg($pf-c-form__select--chevron-svg--Coordinates, $pf-c-form__select--chevron-svg--disabled--Color, $pf-c-form__select--chevron-svg--Width, $pf-c-form__select--chevron-svg--Height);
-    }
-
-    &:hover::-webkit-calendar-picker-indicator,
-    &:focus::-webkit-calendar-picker-indicator {
-      pointer-events: none;
-      opacity: 0;
-    }
-
-    &:-moz-focusring {
-      color: transparent;
-    }
-  }
-
-  // Datalist
-  &__datalist {
-
-    // Disabled
-    &.pf-m-disabled,
-    &:disabled {
-      cursor: not-allowed;
-    }
-  }
-
-  // Input, checkbox, radio, select
-  &__checkbox,
-  &__radio,
-  &__select {
-    cursor: pointer;
-
-    // Disabled
-    &:disabled,
-    &.pf-m-disabled {
-      cursor: not-allowed;
-    }
-  }
-
-  // Checkbox and radio container
-  &__check {
-    // Handle additional elements within __check
-    display: flex;
-    flex: 1;
-    align-items: baseline;
-
-    // Space additional elements within __check
-    > *:not(.pf-c-form__label):not(.pf-c-form__checkbox):not(.pf-c-form__radio) {
-      margin-left: var(--pf-c-form__check--addons--MarginLeft);
-    }
-
-    .pf-c-form__label {
-      // Remove default label padding
-      padding-bottom: 0;
-
-      &:not(.pf-m-disabled):hover {
-        cursor: pointer;
-      }
-
-      &.pf-m-disabled:hover {
-        cursor: not-allowed;
-      }
-    }
   }
 
-  // Checkbox, radio
-  &__checkbox,
-  &__radio {
-    margin-right: var(--pf-c-form__checkbox--MarginRight);
+  // // Checkbox and radio container
+  // &__check {
+  //   // Handle additional elements within check
+  //   display: flex;
+  //   flex: 1;
+  //   align-items: baseline;
+
+  //   // Space additional elements within pf-c-check
+  //   > *:not(.pf-c-form__label):not(.pf-c-check) {
+  //     margin-left: var(--pf-c-form__check--addons--MarginLeft);
+  //   }
+
+  //   .pf-c-form__label {
+  //     // Remove default label padding
+  //     padding-bottom: 0;
+
+  //     &:not(.pf-m-disabled):hover {
+  //       cursor: pointer;
+  //     }
+
+  //     &.pf-m-disabled:hover {
+  //       cursor: not-allowed;
+  //     }
+  //   }
+  // }
+
+  // // Checkbox, radio
+  .pf-c-check {
+    margin-right: var(--pf-c-check--MarginRight);
   }
 
   // Fieldset
   &__fieldset {
     border: none;
 
     .pf-c-form__label {
-      padding-top: var(--pf-c-form__check__label--PaddingTop);
-      padding-right: var(--pf-c-form__check__label--PaddingRight);
-      padding-bottom: var(--pf-c-form__check__label--PaddingBottom);
-      padding-left: var(--pf-c-form__check__label--PaddingLeft);
+      padding-top: var(--pf-c-check-label--PaddingTop);
+      padding-right: var(--pf-c-check-label--PaddingRight);
+      padding-bottom: var(--pf-c-check-label--PaddingBottom);
+      padding-left: var(--pf-c-check-label--PaddingLeft);
```

Comment:
This might be a separate issue from this PR, but these styles look like they're intended for labels used for checkboxes (and radio buttons). But really, they would get applied to any form field that is included inside a `fieldset`, which I don't think was the intention. 

For reference, here is an [article by Léonie Watson on when to use fieldsets](https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/) with some examples.

Reply:
The original intention here was to increase clickable area of a input (checkbox/radio) nested in a label. I scoped it incorrectly..